src/vkscript/command_parser.cc

```diff
@@ -470,13 +470,36 @@ Result CommandParser::ProcessClear() {
 
 Result CommandParser::ParseValues(const std::string& name,
                                   const DatumType& type,
-                                  std::vector<Value>* values) {
+                                  std::vector<Value>* values,
+                                  bool use_std430_layout = false) {
```

Comment:
Remove the = false. I'm not a fan of default values, makes it harder to tell when a param isn't needed anymore.